src/main/java9/module-info.java

```diff
@@ -0,0 +1,19 @@
+module org.xerial.sqlitejdbc {
+
+    requires transitive java.logging;
+    requires transitive java.sql;
+    requires transitive java.sql.rowset;
+
+    exports org.sqlite;
+    exports org.sqlite.core;
+    exports org.sqlite.date;
+    exports org.sqlite.javax;
+    exports org.sqlite.jdbc3;
+    exports org.sqlite.jdbc4;
+    exports org.sqlite.util;
```

Comment:
I'm not sure about this. Can't we export a single module ? We should also probably use a name that is closer to the maven artifact name if that's possible.

Reply:
I am not familiar with Java modules, but since #767 will most likely change the packages, will there be an impact on the modules for the consumers?

for the name i'm afraid `org.sqlite` is too generic. I know the package is currently set to that, but that's also a problem, as there are more Sqlite drivers out there now. You already set the module name to `org.xerial.sqlitejdbc` which i think is fine. The `exports` depend on the existing package, in that case we can't change them until we rename the packages.